src/onnx/onnx.cpp

```diff
@@ -1660,7 +1690,7 @@ struct onnx_parser
             auto&& perm_vals = info.attributes["perm"].ints();
             perm             = std::vector<int64_t>(perm_vals.begin(), perm_vals.end());
         }
-        return mm->add_instruction(migraphx::op::transpose{perm}, args.front());
```

Comment:
for this and L1724, I don't think we need to scope to migraphx when calling make_op (looks to be carried forward from existing code, which doesn't need it either)

Reply:
Actually, I am refactoring the onnx parser, and removed the extra `migraphx::` there.